terasoluna-gfw-functionaltest-web/src/main/resources/ValidationMessages.properties

```diff
@@ -1,7 +1,7 @@
 # this file contains the default messages of JSR 303 bean validation for validations provided in the common library
 
 org.terasoluna.gfw.common.codelist.ExistInCodeList.message = Does not exist in {codeListId}
-
+org.terasoluna.gfw.common.codepoints.ConsistOf.message=Input value is a character type that is not allowed.

```

Comment:
- [ ] change to same message as blank project ``not consist of specified code points``